src/syscall.h

```diff
@@ -8,14 +8,36 @@ See the file LICENSE for details.
 #define SYSCALL_H
 
 #include "kerneltypes.h"
+#include "graphics.h"
 
 #define SYSCALL_exit     1
 #define SYSCALL_testcall 2
 #define SYSCALL_yield    3
 
+#define SYSCALL_window_create 200
+#define SYSCALL_window_set_border_color 201
+#define SYSCALL_window_draw_line 202
+#define SYSCALL_window_draw_arc 203
+#define SYSCALL_window_draw_circle 204
+#define SYSCALL_window_draw_char 205
+#define SYSCALL_window_draw_string 206
```

Comment:
Devil's advocate for both sides here. Indeed, these could all be put into a user library which moves this one level of abstraction up and uses a bunch of calls to a draw_pixel sys_call. This keeps our syscall interface small. At the same time, that could be expensive in terms of performance if we keep entering exiting user mode and putting calls on the function stack. What would Linus do?